src/ppgf.jl

```diff
@@ -59,6 +59,8 @@ using QInchworm.spline_gf: SplineInterpolatedGF,
                            extend!,
                            make_inc_interpolant
 
+import KeldyshED: partition_function
```

Comment:
Importing `KeldyshED.partition_function` here and adding the method `partition_function(P::Vector{<:kd.AbstractTimeGF})::ComplexF64` later leads to [type piracy](https://docs.julialang.org/en/v1/manual/style-guide/#Avoid-type-piracy). Is there a way to avoid that?